src/Frontend/Modules/Location/Widgets/Location.php

```diff
@@ -67,6 +68,8 @@ protected function loadData()
             $this->settings['width'] = $settings['width_widget'];
             $this->settings['height'] = $settings['height_widget'];
             $this->settings['map_type'] = $settings['map_type_widget'];
+            $this->settings['map_style'] = (isset($settings['map_style_widget']))
```

Comment:
you don't need the extra brackets around the isset
also it would be nicer, but I don't know if it won't be to long to have it on one line like

```php
$this->settings['map_style'] = isset($settings['map_style_widget']) ? $settings['map_style_widget'] : 'standard';
```